src/commands/leaderboard.ts

```diff
@@ -0,0 +1,72 @@
+import { CommandInteraction } from "discord.js";
+import { SlashCommandBuilder } from "@discordjs/builders";
+import { RequestError, getLeaderboard, Language, LeaderboardPosition } from "../codewars";
+import { table, TableUserConfig } from "table";
+import { findLanguage, checkBotPlayground } from "../common";
+export { languageAutocomplete as autocomplete } from "../common";
+
+const tableConfig: TableUserConfig = { 
+  drawHorizontalLine: (lineIndex: number, rowCount: number) => lineIndex < 2 || lineIndex === rowCount,
+  columns: [
+    { alignment: 'right' },
+    { alignment: 'left' },
+    { alignment: 'right' },
+    { alignment: 'left' }
+  ],
+};
+
+// leaderboard
+export const data = async () =>
+  new SlashCommandBuilder()
+    .setName("leaderboard")
+    .setDescription("Show the language score leaderboards")
+    .addStringOption((option) =>
+      option
+        .setName("language")
+        .setDescription("The programming language to show leaderboard for")
+        .setAutocomplete(true)
+    )
+    .addIntegerOption((option) =>
+      option.setName("startposition")
+        .setDescription("The top shown position (default 1)")
+        // limits seem not to be checked
+        .setMinValue(1)
```

Comment:
Can you clarify this comment?

Reply:
This setting has no effect. I am still able to input numbers out of that range so there is no enforcement of this limits on Discord side.